docs/style-guide.md

```diff
@@ -0,0 +1,291 @@
+# Rust Style Guide
+
+## Motivation - why not to completely rely a formatting tool?
+
+The docs of rustfmt state that "formatting code is a mostly mechanical task 
+which takes both time and mental effort. By using an automatic formatting tool, 
+a programmer is relieved of this task and can concentrate on more important 
+things.". While in many cases it is true, if the uthor of code does not take 
```

Comment:
Basically the thought of this paragraph is "style is more then just formatting" but we introduce it as if we were to say "formatters don't deliver what they promise". But the quoted statement actually stands true for this case.